integ-tests/build.gradle

```diff
@@ -17,6 +17,8 @@ cucumberTest {
     systemProperties = [
             'aws.region': stack.region.toString(),
             'aws.defaultProfile': stack.profile.toString(),
+
+            'org.slf4j.simpleLogger.log.org.springframework': 'warn',
```

Comment:
Why do we need this line?

Reply:
Ah, I'll add a comment. Along with the dependency on `slf4j-simple`, it suppresses the 10s of lines of debug logs that the Spring context spits out every time that it starts up at the beginning of a scenario.